### ISSUE TITLE
Header and Footer is not printed in mPDF export

### ISSUE THREAD

**Issue by rcalpha** [PHPOffice/PHPExcel#662]
When I save the spreadsheet to PDF using the mPDF library, It does not print the header and footer. 

I exported the to XLSX and Opened the spreadsheet with Office 2013 and It has the header and footer

**Comment by MarkBaker** [comment]
That's right: Headers/Footers aren't implemented in the HTML or PDF Writers

**Comment by dregad** [comment]
> Headers/Footers aren't implemented in the HTML or PDF Writers

@MarkBaker is this by design ? In other words, are you planning to implement header/footer for (m)PDF at some point ? I can understand the logic for not having them in HTML output, but for PDFs, they really should be printed.
